scalableminds/webknossos
# Add boss db dataset

This PR adds an additional tab to the Add Dataset view to add a BossDB dataset. I also updated Antd in the process. I went through the changelog and didn't encounter any breaking changes. I had a look at the admin views and fixed some smaller UI bugs caused by the update.

### URL of deployed dev instance (used for testing):
- https://addbossdbdataset.webknossos.xyz

### Steps to test:
- Set up a wk-connect instance locally (unfortunately there is no wk-connect on our dev instances :/)
- Click on "Add Dataset" and switch to the "Add BossDB Dataset" tab
- Enter the values for a dataset, for example one of these (I'll share credentials with you privately):
```
"domain": "https://api.boss.neurodata.io"
"collection": "ara_2016"
"experiment": "sagittal_50um"

"domain": "https://api.boss.neurodata.io"
"collection": "kasthuri"
"experiment": "kasthuri11"
```
- Test that the validation works and you cannot submit the form without filling in the values.
- When inputting everything correctly, you should be forwarded to the "Import view" and the dataset should show up in the dataset list.

### Issues:
- fixes #3866 

------
~~- [ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~
~~- [ ] Updated [documentation](../blob/master/docs) if applicable~~
~~- [ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes~~
~~- [ ] Needs datastore update after deployment~~
